project/FilteredJacocoAgentPlugin.scala

```diff
@@ -70,17 +70,34 @@ object FilteredJacocoAgentPlugin extends AutoPlugin {
     }
   }
 
+  lazy val Jmf = config("jmf").extend(Compile)
+
   // ---- commands
   private lazy val jacocoCleanAllCmd  = Command.command("jacocoCleanAll") { state =>
     val targets = enabledUnder(state)
     if (targets.isEmpty) { println("[jacoco] nothing to clean (no enabled modules under this aggregate)."); state }
     else targets.foldLeft(state) { (st, ref) => Command.process(s"${ref.project}/jacocoClean", st) }
   }
 
-  private lazy val jacocoReportAllCmd = Command.command("jacocoReportAll") { state =>
-    val targets = enabledUnder(state)
-    if (targets.isEmpty) { println("[jacoco] nothing to report (no enabled modules under this aggregate)."); state }
-    else targets.foldLeft(state) { (st, ref) => Command.process(s"${ref.project}/jacocoReport", st) }
+  private lazy val jacocoReportAllCmd = Command.command("jacocoReportAll") { state0 =>
+    val e         = Project.extract(state0)
+    val current   = e.currentRef
+    // your existing helper (enabled projects under current aggregate)
+    val under     = enabledUnder(state0)
+
+    // Also include current project if enabled
+    val selfEnabled =
+      e.getOpt(current / jacocoPluginEnabled).getOrElse(false)
+
+    val targets = (if (selfEnabled) current +: under else under).distinct
+
+    if (targets.isEmpty) {
+      println("[jacoco] nothing to report (no enabled modules here)."); state0
```

Comment:
```suggestion
      println("[jacoco] nothing to report (no enabled modules here).")
      state0
```

Reply:
Fixed in commit cf0151f8a16adaf444d05d5189b4514d7cd5b259.